ci/storage/test_main_storage_prepare_node.sh

```diff
@@ -15,7 +17,8 @@ if command -v ibv_devinfo; then ibv_devinfo || true; fi
 if lspci | grep NVMe; then
   find /dev -name 'pmem*'
 
-  daos_server storage prepare --scm-only --force
+  # shellcheck disable=SC2086
```

Comment:
Not necessarily for this PR but in the future, I would like to see us getting more comfortable with the safer ways to handle this kind of thing such as using an array rather than disabling the check.